### PR TITLE
Adding separators to "configuring other services"

### PR DESCRIPTION
**Pull request by rickiesherman**
NEW
---
screen shot 2017-04-18 at 12 03 01

OLD
---
screen shot 2017-04-18 at 12 08 43

### CODE REVIEW

**Comment by rickiesherman** [comment]
I also flipped the list to be ordered on the outermost level. The indented lists are unordered.

File: services/databases/_posts/2016-12-12-how-to-clone-your-local-database-for-use-on-runnable.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ description: How to clone your local database for use on Runnable
 * [MySQL](#mysql)
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-clone-your-local-database-for-use-on-runnable.md]: The title for this needs to be in title case

File: services/databases/_posts/2016-12-12-how-to-clone-your-local-database-for-use-on-runnable.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ description: How to clone your local database for use on Runnable
...
 * [PostgreSQL](#postgresql)
 
-## MySQL
+---
+
+### MySQL
 * Take a dump of your MySQL database. The following command will dump the contents of your database to a `sql` file.
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-clone-your-local-database-for-use-on-runnable.md]: Should be ordered list

File: services/databases/_posts/2016-12-12-how-to-clone-your-local-database-for-use-on-runnable.md [hunk 3]
```diff
@@ -63,7 +65,9 @@ description: How to clone your local database for use on Runnable
   -- Dump completed on 2016-09-19 23:38:51
   ```
 
-## PostgreSQL
+---
+
+### PostgreSQL
 * Take a dump of your PostgreSQL database. The following command will dump the contents of your database named `postgres` to a `dump` file.
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-clone-your-local-database-for-use-on-runnable.md]: Should be ordered list

File: services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md [hunk 4]
```diff
@@ -21,15 +21,19 @@ to seed you DB during the build process. This allows you to 'bake' your seed int
...
 
 * [MySQL](#mysql)
 * [PostgreSQL](#postgresql)
 
-# MySQL
-### Upload the seed file to your template configuration
+---
+
+### MySQL
+Upload the seed file to your template configuration
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md]: Colon at the end.

File: services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md [hunk 5]
```diff
@@ -21,15 +21,19 @@ to seed you DB during the build process. This allows you to 'bake' your seed int
 container. Then every branch that you run in isolation will immediately start up with a DB
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md]: Title case heading at the top of this file

File: services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md [hunk 6]
```diff
@@ -39,6 +43,7 @@ Once you have obtained the dump file, continue on the following steps:
 4. Select 'Upload File' to bring up the file selector and choose your dump file (i.e. `seed.sql`).
   ![File Uploaded]({{ site.baseurl }}/images/seed-mysql-db-4.png)
 
+---
 
 ### Enable the database seeding functionality
 1. Uncomment the `ADD seed.sql /seed/sql` line in the Dockerfile editor to the right.
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md]: Empty line needed between list and headings.

File: services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md [hunk 7]
```diff
@@ -39,6 +43,7 @@ Once you have obtained the dump file, continue on the following steps:
 4. Select 'Upload File' to bring up the file selector and choose your dump file (i.e. `seed.sql`).
   ![File Uploaded]({{ site.baseurl }}/images/seed-mysql-db-4.png)
 
+---
 
 ### Enable the database seeding functionality
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md]: All headings need to be title case

File: services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md [hunk 8]
```diff
@@ -47,9 +52,10 @@ Once you have obtained the dump file, continue on the following steps:
     Make sure you leave the second `seed.sql` intact so the init script will work.
 2. Now click `Save & Build` to build your container and seed your database.
 
+---
 
-# PostgreSQL
-### Upload the seed file to your template configuration
+### PostgreSQL
+Upload the seed file to your template configuration
```
Comment (runnabro) [inline services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md]: Colon + line break between list and heading

**Pull request merged**